### ISSUE TITLE
EPG for all channels

### ISSUE THREAD

**Issue by joaquinito2070** [dtankdempse/daddylive-m3u#15]
EPG IS missing, for example, spanish channels, among others.

**Comment by dtankdempse** [comment]
Yes, they have not been added yet. I am still working on looking for guide information for the other channels.

**Comment by Lancs-Pete** [comment]
This is a very good source for epg

https://www.bevy.be/epg-guide/

**Comment by dtankdempse** [comment]
> This is a very good source for epg
> 
> https://www.bevy.be/epg-guide/

Yes, I was already looking into that site, but the last time I checked, they were displaying incorrect guide data. I'll take another look. Thanks.

**Comment by Lancs-Pete** [comment]
Sometimes I think the data is not updated till mudlle of day (CET Timezone)
but it's pretty good normally
